start-mycroft.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
we should still be using `env`
```
#!/usr/bin/env sh
```

Reply:
I don't see a reason why. For one, env is a GNU Coreutils thing, and I'd like this to work on systems without those installed. Secondly, we can assume that on POSIX systems `/bin/sh` is always available and it's a POSIX compatible shell. If it's not, that's broken on the distro in question and not something that we should care about, as that's just a system not being POSIX.

Reply:
sure thing! sounds good